arcgis-ios-sdk-samples/Layers/Blend renderer/BlendRendererViewController.swift

```diff
@@ -49,7 +49,7 @@ class BlendRendererViewController: UIViewController, BlendRendererSettingsVCDele
         // Dispose of any resources that can be recreated.
     }
     
-    private func generateBlendRenderer(altitude: Double, azimuth: Double, slopeType: AGSSlopeType, colorRampType: AGSPresetColorRampType) -> AGSBlendRenderer {
+    fileprivate func generateBlendRenderer(_ altitude: Double, azimuth: Double, slopeType: AGSSlopeType, colorRampType: AGSPresetColorRampType) -> AGSBlendRenderer {
```

Comment:
similar to other comments above about having argument name to clarify call site usage